charts/dispatch/values.yaml

```diff
@@ -15,5 +15,15 @@ global:
   port: 443
   tls:
     secretName: dispatch-tls
+  db:
+    backend: postgres
+    host: postgresql
+    port: 5432
+    user: vmware
+    password: dispatch
+    database: dispatch
+    # namespace: <set if different from dispatch>
```

Comment:
If these are settable values, it's nice to actually put the default in here.  Otherwise remove.

Reply:
it's some leftovers, but good catches, I will fix it.